src/components/SecondaryPanes/Breakpoints/Breakpoint.js

```diff
@@ -125,18 +122,14 @@ class Breakpoint extends PureComponent<Props> {
 
   highlightText() {
     const text = this.getBreakpointText();
-    const sourceEditor = getEditor();
+    const codeMirror = getCodeMirror();
 
-    if (!text || !sourceEditor || !sourceEditor.editor) {
+    if (!text || !codeMirror) {
       return { __html: "" };
```

Comment:
Due to https://github.com/devtools-html/debugger.html/issues/6439, I'm thinking we should consider showing the escaped text in the case that there's no text or codemirror.

Reply:
hmm, i wonder if this was with the first mapped breakpoints PR and not pure components? do we see it now with the latest 61 release in nightly?

Reply:
thinking of this 
https://github.com/devtools-html/debugger.html/blob/master/src/selectors/breakpointSources.js#L28-L31

Reply:
I see... we should change this strategy so that we just get the CodeMirror class, we don't really need the instance around to do it.

Reply:
I think I can fix #6439 in a bit